src/main/java/net/zerowatch/OAuthCommand.java

```diff
@@ -0,0 +1,82 @@
+package net.zerowatch;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Handles oAuth requests.
+ */
+ @RestController
+public final class OAuthCommand {
```

Comment:
Whenever you install an app to your Slack team, there's an oAuth dance going on so that your app gets credentials for that team. Only when the process is complete is the app considered installed and its commands/bots made available to the team.

See https://github.com/pylaligand/overwatch-slack-bot/issues/5 for a follow-up.